week06_deployment/README.md

```diff
@@ -0,0 +1,11 @@
+# Deployment
+
+See README files inside directories for more info about how to run stands.
```

Comment:
Also, please mention the home assignment subfolder somewhere here (maybe at the end of file, after the description of the practice stands)